src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10109,8 +10103,9 @@ void Player_UpdateCommon(Player* this, GlobalContext* globalCtx, Input* input) {
 
         if ((globalCtx->csCtx.state != 0) && (this->csMode != 6) && !(this->stateFlags1 & 0x800000) &&
             !(this->stateFlags2 & 0x80) && (this->actor.type == ACTORTYPE_PLAYER)) {
+            CsCmdActorAction *linkAction = globalCtx->csCtx.linkAction;
```

Comment:
I believe this temp is the only change necessary to match this function, unless I'm missing something. 
Also I suggest naming the temp something more descriptive like `linkActionCsCmd`, but that doesn't really matter. 
And make sure to run the formatter after all changes so things like the pointer star style here gets fixed for you.

Reply:
I think there are still some changes in this function that could be omitted (eg. the globalCtx2 stuff and other temp changes)

Reply:
you're right, the temp changes got it closer to matching, but I guess the CsCmdActorAction temp fixed everything at once, even making everything else unnecessary.